### ISSUE TITLE
logs arent grep/awk friendly

### ISSUE THREAD

**Issue by avsm** [hannesm/tlstunnel#13]
I occasionally see exceptions in there as well, would be good if those could be separated or logged in the same field structure.

e.g.

```
[14:15:01] 65.112.10.251:61305: connection closed read 0 bytes, wrote 0 bytes
[14:15:01] 65.112.10.251:61306: connection closed read 0 bytes, wrote 0 bytes
[14:15:01] 65.112.10.251:61307: connection closed read 0 bytes, wrote 0 bytes
async error End_of_file
[14:16:09] 173.252.73.114:49799: connection established (TLS version 1.1, TLS_DHE_RSA_WITH_AES_256_CBC_SHA)
[14:16:10] 173.252.73.114:49799: connection closed read 195 bytes, wrote 73166 bytes
[14:16:10] 91.227.212.3:29903: connection established (TLS version 1.2, TLS_DHE_RSA_WITH_AES_256_CBC_SHA)
[14:16:10] 91.227.212.3:29957: connection established (TLS version 1.2, TLS_DHE_RSA_WITH_AES_256_CBC_SHA)
[14:16:10] 91.227.212.3:29958: connection established (TLS version 1.2, TLS_DHE_RSA_WITH_AES_256_CBC_SHA)
```

**Comment by hannesm** [comment]
agreed. the code should be carefully rewritten in a way that the `async_exception_hook` is never triggered, but all exceptions inside of the `worker` are handled properly

**Comment by hannesm** [comment]
this is solved afaics, or do you see after e3576df15e5faf3c24dd875740dff4de2efb2891 async exceptions? if so, please reopen
